ee/clickhouse/queries/test/test_paths.py

```diff
@@ -1866,7 +1866,7 @@ def test_path_removes_duplicates(self):
             ],
         )
 
-    @test_with_materialized_columns(["$current_url"])
+    @test_with_materialized_columns(["$current_url", "$screen_name"])
```

Comment:
random Paths failing test, I'm guessing used to pass earlier because no proper teardown earlier on materialized columns, and some new truncating on events table by @EDsCODE ?